app/models/component.rb

```diff
@@ -20,15 +22,18 @@ class Component < Aggregate
         -> { where(component_type: 'VSP').or(where(component_type: 'SP')) }
 
   def self.to_service_metadata(event_id, published_at = Time.now)
-    matching_service_adapters = Component.matching_service_adapters
-                                         .includes(:enabled_signing_certificates,
-                                                   :encryption_certificate)
-    service_providers = Component.service_providers
-                                 .includes(:enabled_signing_certificates,
-                                           :encryption_certificate)
-    { published_at: published_at, event_id: event_id,
+    matching_service_adapters = MsaComponent.matching_service_adapters
```

Comment:
do we still need the scopes above since `MsaComponent.service_providers` will no longer be valid?